projects/cm_mcu/clocksynth.c

```diff
@@ -128,13 +128,14 @@ static int write_register(int RegList[][2], int n_row)
     }
     HighByte = NewHighByte;
     uint8_t LowByte = RegList[i][0] - (NewHighByte << 8);
-    
+    uint16_t LowByte_reg_addr = LowByte;
+    uint16_t CHANGEPAGE_REG_ADDR = 0x01;
```

Comment:
please make this a #define instead of a regular variable (it's a constant, and that is the convention in the rest of the code.)

Reply:
fixed it.